TwoPage/src/androidTest/java/com/microsoft/device/display/samples/twopage/TwoPageModeOrientationTest.kt

```diff
@@ -31,7 +31,7 @@ class TwoPageModeOrientationTest {
     private val device = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
 
     @get:Rule
-    val activityRule = ActivityTestRule<MainActivity>(MainActivity::class.java)
+    val activityRule = ActivityTestRule<TwoPageActivity>(TwoPageActivity::class.java)
```

Comment:
Are you going to do the same regarding replacing `ActivityTestRule` like previous PRs? I think we should ;-)

Reply:
I already did this in another commit, please check.